examples/motoko_examples/quicksort/src/motoko_base.ts

```diff
@@ -0,0 +1,29 @@
+import {
+    int
+} from 'azle';
+
+export const Int = {
+    abs: Math.abs,
```

Comment:
Let's also pull these out as named functions with the `function` keyword syntax

Reply:
I've pulled out the `compare` function and didn't see any other offenses in the rest of the codebase. This should be good now.